package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-sdk",
-  "version": "7.10.0",
+  "version": "7.10.1",
```

Comment:
The PR contain a very important breaking change, and will cause very important problems on all projects that use kuzzle with http if they upgrade their version without reading release note. 
And that is bad with a PATCH version. 
The only way to follow the semvers standard is to increment the major version or to increment minor and authorize PUT and PATCH (to avoir breaking change)